pkg/client/resmgmt/resmgmt_test.go

```diff
@@ -967,7 +967,7 @@ func TestCCProposal(t *testing.T) {
 	rc := setupResMgmtClient(t, ctx)
 
 	ccPolicy := cauthdsl.SignedByMspMember("Org1MSP")
-	instantiateReq := InstantiateCCRequest{Name: "name", Version: "version", Path: "path", Policy: ccPolicy}
+	instantiateReq := InstantiateCCRequest{Name: "name", Version: "version", Lang: pb.ChaincodeSpec_GOLANG, Path: "path", Policy: ccPolicy}
```

Comment:
Should revert this, as this PR shouldn't break the old client code.

Reply:
OK, Fixed.